packages/gensx-storage/src/utils/parse-error.ts

```diff
@@ -0,0 +1,11 @@
+export async function parseErrorResponse(response: Response): Promise<string> {
+  try {
+    const data = (await response.json()) as { error?: string };
```

Comment:
If the caller eventually tries to call `response.json()` on this, it will error because the body was already read. Safer to not mutate the input and clone the response.

```suggestion
  try {
    const clone = response.clone();
    const data = (await clone.json()) as { error?: string };
```

Reply:
good call, updated